### PR TITLE
small fixes chg

### PR DESCRIPTION
**Pull request by mhajij**
fixing update node attrs  in CHG.

### CODE REVIEW

File: toponetx/classes/cell_complex.py [hunk 1]
```diff
@@ -2234,10 +2234,8 @@ def get_linegraph(self, s=1, cells=True):
         nx.Graph
             A NetworkX graph representing the s-linegraph of the Cell Complex.
         """
-        if isinstance(s, None):
-            ValueError(
-                "s must be a positive integer larger than 1, got type of s None."
-            )
+        if not isinstance(s, int):
+            ValueError("s must be a positive integer larger than 1.")
```
Comment (ffl096) [inline toponetx/classes/cell_complex.py:2238]: The check and error message do not align. It is not checked whether `s` is larger than 1, just that it is an int.

Comment (mhajij) [reply toponetx/classes/cell_complex.py]: fixed

File: toponetx/classes/cell_complex.py [hunk 2]
```diff
@@ -2233,11 +2233,19 @@ def get_linegraph(self, s=1, cells=True):
         -------
         nx.Graph
             A NetworkX graph representing the s-linegraph of the Cell Complex.
+
+        Examples
+        --------
+        >>> CC = CellComplex()
+        >>> CC.add_cell([0,1,2,3,4],rank=2)
```
Comment (ffl096) [inline toponetx/classes/cell_complex.py:2240]:
```suggestion
        >>> CC.add_cell([0, 1, 2, 3, 4], rank=2)
```

File: toponetx/classes/cell_complex.py [hunk 3]
```diff
@@ -2233,11 +2233,19 @@ def get_linegraph(self, s=1, cells=True):
...
+        >>> G = CC.get_linegraph()
         """
-        if isinstance(s, None):
-            ValueError(
-                "s must be a positive integer larger than 1, got type of s None."
-            )
+        if not isinstance(s, int) or s < 1:
+            raise TopoNetXError(f"'s' must be a positive integer, got {s}.")
```
Comment (ffl096) [inline toponetx/classes/cell_complex.py:2244]:
```suggestion
            raise ValueError(f"`s` must be a positive integer, got {s}.")
```

File: toponetx/classes/cell_complex.py [hunk 4]
```diff
@@ -2233,11 +2233,19 @@ def get_linegraph(self, s=1, cells=True):
...
+
+        if not isinstance(cells, bool):
+            raise TopoNetXError(f"'cells' must be a boolean, got {cells}.")
```
Comment (ffl096) [inline toponetx/classes/cell_complex.py:2247]:
```suggestion
            raise TypeError(f"'cells' must be a boolean, got {cells}.")
```

**Review by ffl096 — CHANGES_REQUESTED** [review]
Minor comments, otherwise this is good to go!

**Review by ffl096 — APPROVED** [review]

**Pull request merged**